brave/brave-core
# Fix #18918 - Null deref in BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH

Resolves https://github.com/brave/brave-browser/issues/18918

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Enable media router in brave://settings/extensions
2. Enable all cast flags in brave://cast
3. Navigate to https://kirtikumarar.com/brave/cast.html and click cast
4. Should not crash
